### PR TITLE
109 - ODS Component - Small card

### PR DESCRIPTION
**Pull request by paulinea**

### CODE REVIEW

File: app/lib/ui/components/cards/card_small.dart [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  const CardSmall({super.key});
+
+  @override
+  State<CardSmall> createState() => _CardSmallState();
+}
+
+class _CardSmallState extends State<CardSmall> {
+  @override
+  Widget build(BuildContext context) {
+    var cardWidth = -2 * spacingM;
```
Comment (ludovic35) [inline app/lib/ui/components/cards/card_small.dart:16]: Add a comment to explain why

File: app/lib/ui/components/cards/card_small.dart [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    return Scaffold(
+      backgroundColor: Colors.white,
+      appBar: AppBar(
+        title: Text('Small Card'),
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(spacingM),
+        child: Column(
+          children: <Widget>[
```
Comment (ludovic35) [inline app/lib/ui/components/cards/card_small.dart:31]: no need <Widget> here

File: lib/lib/components/card/ods_small_card.dart [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            borderRadius: BorderRadius.circular(4.0),
+          ),
+          elevation: 1,
+          clipBehavior: Clip.antiAliasWithSaveLayer,
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              SizedBox(
+                width: double.infinity,
+                height: 110,
```
Comment (ludovic35) [inline lib/lib/components/card/ods_small_card.dart:48]: why 110 here ?

**Review by ludovic35 — APPROVED** [review]

File: app/lib/ui/components/cards/card_small.dart (hunk 2 above)
Comment (paulinea) [reply app/lib/ui/components/cards/card_small.dart:31]: If I remove the column, the card takes full screen height. 
I don't find any other solution: https://stackoverflow.com/questions/42257668/the-equivalent-of-wrap-content-and-match-parent-in-flutter

**Pull request merged**